.github/workflows/validate.yml

```diff
@@ -63,6 +63,9 @@ jobs:
 
   archutil-arm64:
     runs-on: ubuntu-22.04
+    # TODO: enable when binutils-loongarch64-linux-gnu pkg is available for aarch64 arch
+    # https://github.com/moby/buildkit/pull/4392#issuecomment-1938223235
+    if: false
```

Comment:
@yzewei As discussed with @tonistiigi I think we can disable this job in the meantime.